modules/rest-api/pages/rest-xdcr-identify-target-cluster.adoc

```diff
@@ -0,0 +1,180 @@
+= Identifying the Target in Replication
+:description: pass:q[]
+
+[abstract]
+{description}
+
+== HTTP Method and URI
+
+----
+GET /xdcr/sourceClusters
+----
+
+== Description
+
+When on a current cluster, XDCR lets you:
```

Comment:
Instead of
When on a current cluster, XDCR lets you:
Replace with
When on a cluster, XDCR lets you: